core/Cookie.php

```diff
@@ -205,14 +206,15 @@ public function save($sameSite = null)
      * Extract signed content from string: content VALUE_SEPARATOR '_=' signature
      *
      * @param string $content
+     * @param bool $validate
      * @return string|bool  Content or false if unsigned
      */
-    private function extractSignedContent($content)
+    private function extractSignedContent($content, $validate = true)
```

Comment:
Not too important but maybe the parameter doesn't need to have a default value?

Reply:
As this function is private, it makes sense :+1:

Reply:
if you have time be good to change otherwise not too important